code/web/src/modules/survey/Survey/Item.js

```diff
@@ -28,9 +28,8 @@ class Item extends PureComponent {
   }
 
   handleItemClick = () => {
-    console.log(this.state.selected)
     this.setState({ selected: !this.state.selected });
-    updateSelectedItems(this.props.item, this.state.selected)
+    this.props.updateSelectedItems(this.props.item, !this.state.selected, this.props.page)
```

Comment:
this is a critical piece to getting updateSelectedItems working with Redux. Thanks